support/Tests/GraphQLSupportTests/IntegrationTests.swift

```diff
@@ -56,6 +56,24 @@ class IntegrationTests: XCTestCase {
         XCTAssertEqual(queryString, "mutation{set_integer(input:{key:\"answer\",value:42,negate:true})}")
     }
 
+    func testInputObjectExplictNilConstructor() {
+        let query = Generated.buildMutation { $0
+            .setInteger(input: Generated.SetIntegerInput(key: "answer", value: 42, negate: .some(nil)))
```

Comment:
Yes, I _want_ something that you unwrap to nil. The problem is distinguishing between: `foo(optinal: nil`) and `foo()` which I can't do at a language level because within `foo`, `optional` will have a value of nil regardless of whether the caller provided the nil explicitly or not.